app/src/androidTest/java/com/deque/mobile/axedevtoolssampleapp/InstrumentationRegistryExampleTest.kt

```diff
@@ -15,52 +20,55 @@ class InstrumentationRegistryExampleTest {
     @JvmField
     val rule: ActivityScenarioRule<MainActivity> = ActivityScenarioRule(MainActivity::class.java)
 
-
     private val axe = AxeDevTools()
 
     init {
         axe.loginWithApiKey(BuildConfig.AXE_DEVTOOLS_APIKEY)
-    }
+         }
 
     @Before
     fun setupAxeDevTools() {
+        axe.tagScanAs(setOf("myAxeEspressoTest"))
+        axe.setTestingConfig(
+            AxeDevToolsEspressoConfig(IdlingRegistry.getInstance())
+        )
         axe.setInstrumentation(InstrumentationRegistry.getInstrumentation())
     }
 
     @Test
-    fun test_using_registry() {
-        axe.setInstrumentation(InstrumentationRegistry.getInstrumentation())
-        val scanResultHandler = axe.scan()
-//      1. Upload it to the dashboard
```

Comment:
Recommendation: The purpose of these comments is to guide the user to choose from different options. So I would recommend keeping this comment.